lib/questionnaires/view/src/questionnaire_stepper_page_view.dart

```diff
@@ -168,6 +177,11 @@ class QuestionnaireStepperPageViewController {
       return;
     }
 
+    final currentPage = _state?._pageController.page?.round();
+    if (currentPage == null) {
+      return;
+    }
```

Comment:
Maybe we can put a small comment here about why checking if `currentPage` is null is needed.

Also, not sure, but if all we need is to check if the value is non-null or not, maybe there's no need to call `round`?

Reply:
Sorry, it's other unintended changes.  I will revert them now. 🙇